.github/scripts/run-scala-test.py

```diff
@@ -16,7 +16,7 @@ def run_scala_test(target_project, test_name):
     with cd(manager_fsim_dir), prefix('source env.sh'):
         # avoid logging excessive amounts to prevent GH-A masking secrets (which slows down log output)
         with settings(warn_only=True):
-            rc = run("make -C sim testOnly TARGET_PROJECT={} SCALA_TEST={} &> scala-test.full.log".format(target_project, test_name)).return_code
+            rc = run(f"make -C sim testOnly TARGET_PROJECT={target_project} SCALA_TEST={test_name} ENABLE_SBT_THIN_CLIENT=1 &> scala-test.full.log").return_code
             if rc != 0:
```

Comment:
Remind me why we cannot always print the standard out? If we must write to a file to control the volume produced, can we tail the end of the log so that we get ScalaTest's summary. 

This so we can be sure the correct things are running (see: FPGA-deployment logs).

Reply:
Printing anything large is an issue because GH-A has to parse the output and remove secrets. If there are alot of secrets (or big secrets like the `.pem` file) then the CI slows down significantly.

Reply:
ugggh why, GHA, why. Ok -- tail it please.